lib/iiif_manifest/manifest_builder/structure_builder.rb

```diff
@@ -9,7 +9,7 @@ def initialize(record, canvas_builder_factory:, iiif_range_factory:)
       end
 
       def apply(manifest)
-        top_ranges.each_with_index do |top_range|
+        top_ranges.each do |top_range|
```

Comment:
Should this have been a different pull request?  This looks like a good fix, but I'm not sure how it relates to the metadata stuff.

Reply:
done